src/sorcha/utilities/sorchaConfigs.py

```diff
@@ -0,0 +1,1272 @@
+from dataclasses import dataclass
+import configparser
+import logging
+import sys
+import os
+import numpy as np
+from sorcha.lightcurves.lightcurve_registration import LC_METHODS
+from sorcha.activity.activity_registration import CA_METHODS
+
+
+@dataclass
+class inputConfigs:
+    """Data class for holding INPUTS section configuration file keys and validating them."""
+
+    ephemerides_type: str = ""
```

Comment:
Meg has convinced me that all defaults should be None, so:

`ephemerides_type: str = None`

This makes it clearer that the attribute is unpopulated. Empty strings and zeros do evaluate as False in Python, but they can also cause confusion - especially when a zero (or an empty string) is a legitimate, wanted value.